index.html

```diff
@@ -0,0 +1,129 @@
+            <ul class="goals_content">
+              <li>Get more knowledge about HTML, JavaScript и CSS</li>
+              <li>Get enough practice for site layout</li>
+              <li>Pass the interview and get the opportunity to start working in EPAM  </li>
+              <li>To achieve goals I:<ul>
+                <li>Allocate a decent time every day to learn</li>
+                <li>Practice every day</li>
+                <li>Viewing useful resources and Livecodding sessions</li></ul> </li>
+            </ul>
+            <div class="vertical_rectangle"></div>
+          </div>
+          <div class="skills">
+            <div class="underscore2">
+              <p class="underscore_content2"><a name="skills">Skills</a></p>
+              <div class="rectangle2"></div>
+            </div>
+            <ul class="skills_content">
+              <li>С++ basics</li>
+              <li>Java basics</li>
+              <li>HTML/CSS basics </li>
+              <li>SQL</li>
+            </ul>
+            <div class="vertical_rectangle2"></div>
+          </div>
+          <div class="education">
+            <div class="underscore3">
+              <p class="underscore_content3"><a name="education">Education</a></p>
+              <div class="rectangle3"></div>
+            </div>
+            <ul class="education_content">
+              <li>At this moment I am second-year student of BSUIR (FITU)</li>
+              <li>Also I completed several courses from Codeacademy, such as<ul>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>SQL</li></ul></li>
+              <li>Courses in Java</li>
+              <li>English level: B1
+              (11 years in gymnasium + 1 year in the University)</li>
+            </ul>
+            <div class="vertical_rectangle3"></div>
+          </div>
+          <div class="experience">
+            <div class="underscore4">
+              <p class="underscore_content4"><a name="experience">Experience</a></p>
+              <div class="rectangle4"></div>
+            </div>
+            <ul class="experience_content">
+              <li>Unfortunately, I have not got any experience in this area</li>
+            </ul>
+            <div class="vertical_rectangle4"></div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer_content">
+        <div class="footer_text">
+          <p class="student1">Ivanova Aleksandra</p>
+          <p class="phone">+375447****32</p>
+        </div>
+        <div class="contact_links">
+          <ul class="links">
+            <li><a href="https://vk.com/id146633339"><i class="fab fa-vk"></i></a></li>
+            <li><a href="https://www.instagram.com/lek_an_arteg/"><i class="fab fa-instagram"></i></a></li>
+            <li><a href="https://github.com/leqsar"><i class="fab fa-github"></i></a></li>
+          </ul>
+        </div>
+      </div>
+    </footer>
+  </body>
+</html>
```

Comment:
use more reasonable class names, code styling should be more consistant, it would be cool if you use more semantic tags and more attention for section tag. Do not use additional tags